### PR TITLE
test: Account osd sizing in add capacity tests

### PR DESCRIPTION
**Pull request by afreen23**
- current tests are breaking due to the ignorance of osd sizing while adding capacity

cc @ebondare

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/add-capacity.scenario.ts [hunk 1]
```diff
@@ -63,8 +64,8 @@ describe('Check add capacity functionality for ocs service', () => {
     beforeAll(async () => {
       [expansionObjects.clusterJSON] = storageCluster.items;
       expansionObjects.name = getName(expansionObjects.clusterJSON);
-      expansionObjects.previousCnt =
-        expansionObjects.clusterJSON?.spec?.storageDeviceSets?.[0]?.count;
+      const deviceSet = expansionObjects.clusterJSON?.spec?.storageDeviceSets?.[0];
+      expansionObjects.previousCnt = deviceSet?.count;
```
Comment (cloudbehl) [inline frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/add-capacity.scenario.ts:68]:
```suggestion
      const deviceSets = expansionObjects.clusterJSON?.spec?.storageDeviceSets;
      expansionObjects.previousCnt = deviceSets?.[0]?.count;
```

Comment (cloudbehl) [reply frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/add-capacity.scenario.ts:68]: or probably we can call it `firstDeviceSet` or `initialDeviceSet`

**Review by cloudbehl — COMMENTED** [review]
nit:

File: frontend/packages/ceph-storage-plugin/integration-tests/views/add-capacity.view.ts [hunk 2]
```diff
@@ -33,11 +33,13 @@ export const currentACSelector = (() => {
   }
 })();
 
-export const verifyFields = async () => {
+export const verifyFields = async (size) => {
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/integration-tests/views/add-capacity.view.ts:36]:
```suggestion
export const verifyFields = async (size: number) => {
```

**Review by bipuladh — COMMENTED** [review]
nit

File: frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/add-capacity.scenario.ts [hunk 3]
```diff
@@ -63,8 +64,8 @@ describe('Check add capacity functionality for ocs service', () => {
     beforeAll(async () => {
       [expansionObjects.clusterJSON] = storageCluster.items;
       expansionObjects.name = getName(expansionObjects.clusterJSON);
-      expansionObjects.previousCnt =
-        expansionObjects.clusterJSON?.spec?.storageDeviceSets?.[0]?.count;
+      const initialDeviceSet = expansionObjects.clusterJSON?.spec?.storageDeviceSets?.[0];
```
Comment (a2batic) [inline frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/add-capacity.scenario.ts:67]: nit
```suggestion
      const initialDeviceSet: DeviceSet = expansionObjects.clusterJSON?.spec?.storageDeviceSets?.[0];
```

**Pull request merged**